src/sentry/static/sentry/app/components/modals/helpSearchModal.jsx

```diff
@@ -0,0 +1,71 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled, {css} from 'react-emotion';
+
+import {t} from 'app/locale';
+import Search from 'app/components/search';
+import HelpSource from 'app/components/search/sources/helpSource';
+import theme from 'app/utils/theme';
+
+const dropdownStyle = css`
+  width: 100%;
+  border: transparent;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  position: initial;
+  box-shadow: none;
+  border-top: 1px solid ${p => theme.borderLight};
+`;
+
+class HelpSearchModal extends React.Component {
+  static propTypes = {
+    Body: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
```

Comment:
Uppercase prop seems unusual compared to other components we have.

Reply:
Yeah this is weird, will fix.

Reply:
Wait nevermind, it is because it's a component.

We do the same thing in the command palette modal

https://github.com/getsentry/sentry/blob/3e970700c5c254c2363a1de6429e125da69bea35/src/sentry/static/sentry/app/components/modals/commandPalette.jsx#L21-L25

Reply:
Yeah this is something I added for parent components to inject components so that we could have some consistent UI - does it make sense to do this?